Library/Homebrew/requirements/cctools_requirement.rb

```diff
@@ -0,0 +1,8 @@
+class CctoolsRequirement < Requirement
+  fatal true
+  default_formula 'cctools'
+
+  satisfy do
+    MacOS::Xcode.installed? || MacOS::CLT.installed?
```

Comment:
Since `cctools` is already the default formula, I don't think we need an explicit check for it in the `satisfy` block.